azkaban-web-server/src/main/java/azkaban/webapp/Status.java

```diff
@@ -29,18 +29,21 @@
 public class Status {
 
   private final String version;
+  private final String pid;
   private final String installationPath;
   private final long usedMemory, xmx;
   private final boolean isDatabaseUp;
   private final Map<Integer, Executor> executorStatusMap;
 
   Status(final String version,
+      final String pid,
```

Comment:
Looks like this pid is for Web server? Is that necessary to monitor? I believe executors' pid even deserves being monitored.